math/libfaiss/Portfile

```diff
@@ -24,39 +22,134 @@ long_description    Library for efficient similarity search and clustering of \
                     with complete wrappers for Python/numpy. Some of the most \
                     useful algorithms are implemented on the GPU.
 
+checksums           rmd160  3b1180dc7ceadc484bb86cfe32fb5cd1c4a694b9 \
+                    sha256  6962641a9183e12d394aa2f9c443593734a373d60c61fd0633edda5029d2fa78 \
+                    size    2762193
+
 compilers.choose    cxx
 # The Apple native llvm compiler does not support OpenMP.
 compiler.blacklist  llvm-gcc-4.2
-compiler.fallback   macports-gcc-7
+compiler.fallback   macports-gcc-9
```

Comment:
this doesn't seem right... for example if I do `port info libfaiss +gcc10` I both get `gcc9` and `gcc10` as build dependencies. I am not completely sure how this is supposed to work and why it's needed, but I don't think it does what you want right now

Reply:
I don’t know either. Should these lines be deleted?

Reply:
The blacklist can be deleted. The fallback is necessary for the build. Not sure why.